tests/codestyle/fixtures/noUselessParenthesisFixer/actual.php

```diff
@@ -2,7 +2,6 @@
 
 declare(strict_types=1);
 
-if (($a === 1))
-{
+if (($a === 1)) {
```

Comment:
Why did you change this?

Reply:
Because this fixer is responsible for fixing a different thing.